### ISSUE TITLE
Bug in neutral input/interpolation code

### ISSUE THREAD

**Issue by mattzett** [gemini3d/gemini3d#30]
Somehow the program is interpreting the neutral data as smaller than it should be...  This problem does not manifest in commit a60e56a so it was introduced sometime in the last eight months probably during one of the neutral refactors...

**Comment by mattzett** [comment]
This bug only manifests with axisymmetric interpolation.  It could be an issue with rho, z getting mixed up with the input or some issue with .nml vs. .ini input.  Need to check.

**Comment by mattzett** [comment]
Running with show console output:

```
Neutral data has lhorzn,lz size:           330         900  with spacing dhorzn,dz   2000.0000000000000        2000.0000000000000
```

which appears to indicate that the horizontal and vertical dimensions are getting permuted accidentally...

**Comment by mattzett** [comment]
I'm trying a workaround for simsize3 that basically swaps the horizontal and vertical dimensions to see if this correct things...

**Comment by mattzett** [comment]
This has been fixed here:  b30b1c2
